plugins/woocommerce/src/StoreApi/Authentication.php

```diff
@@ -22,6 +23,21 @@ public function init() {
 		add_filter( 'rest_pre_serve_request', array( $this, 'send_cors_headers' ), 10, 3 );
 		add_filter( 'rest_allowed_cors_headers', array( $this, 'allowed_cors_headers' ) );
 
+		// If cart has a valid token, override the core session class.
```

Comment:
These changes were necessary to load cart data early in the request!

Reply:
cc @wavvves

Reply:
I don't there's a problem, actually believe this cart token thing is better suited to be here on Auth than the Abstract route 👍🏼